Assets/_Project/Scripts/Map/MapStyleInitializer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Mapbox.Unity.Map;
+using Mapbox.Unity.Utilities;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.Map
+{
+    public class MapStyleInitializer 
+    {
+        public void SetMapImageLayer(AbstractMap map, string styleURL, string latLong, int zoom)
+        {
+            map.ImageLayer.SetProperties(ImagerySourceType.Custom, true, false, false);
```

Comment:
Consider extracting those two method calls (SetProperties and SetLayerSource) to aa separate private method. DRY.